### PR TITLE
chore: bump version number for minor fix

### PR DESCRIPTION
**Pull request by MCBrandenburg**
Noticed that the build version wasn't updated, so updated and bumped the minor up

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "watch",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (stephenmathieson) [inline package.json:3]: This was correct. v0.4.1 doesn't exist yet.

File: src/watch.c [hunk 2]
```diff
@@ -18,7 +18,7 @@
  * Command version.
  */
 
-#define VERSION "0.3.1"
+#define VERSION "0.4.1"
```
Comment (stephenmathieson) [inline src/watch.c:21]:
```suggestion
#define VERSION "0.4.0"
```

**Review by stephenmathieson — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (MCBrandenburg) [reply package.json:3]: Wouldn't this fix cause a new release?

**Comment by MCBrandenburg** [comment]
Made the change in `src/watch.c` and removed the change to package.json

File: package.json (hunk 1 above)
Comment (stephenmathieson) [reply package.json]: No, there's a separate release process I'll run. It updates this, the changelog, creates a release commit and git tag.

Comment (MCBrandenburg) [reply package.json]: Thank you!

**Pull request merged**